x-pack/plugins/fleet/server/services/agents/crud.ts

```diff
@@ -157,6 +158,9 @@ export async function getAgentTags(
   if (showInactive === false) {
     filters.push(ACTIVE_AGENT_CONDITION);
   }
+  if (!kuery?.toLowerCase().includes('status:*')) {
```

Comment:
should we also exclude that condition for `status:unenrolled` ?

Reply:
done